tools/run_tests_docker.sh

```diff
@@ -149,8 +150,13 @@ if test ${special}; then
     echo 'Using the special config and only running test_dirac'
     pytest -v --full-trace lib/rucio/tests/test_dirac.py
 else
-    echo 'Running tests'
-    pytest -v --full-trace
+    if test ${notrace}; then
+	echo 'Running tests'
+	pytest -v --tb=short
+    else
+	echo 'Running tests with full tracing'
+	pytest -v --full-trace
```

Comment:
I have already removed the "full-trace" for regular test runs. I realized the normal output also contains the trace without code output for each stack frame. And I agree it was too much output to have a good overview.